src/helpers/helpers.canvas.js

```diff
@@ -44,112 +44,126 @@ var exports = module.exports = {
 		}
 	},
 
-	drawPoint: function(ctx, style, radius, x, y) {
-		var type, edgeLength, xOffset, yOffset, height, size;
+	drawPoint: function(ctx, style, width, fontSize, x, y, isLineWidthZero) {
```

Comment:
I don't knw that `fontSize` should be a param here. `height` makes a lot more sense.